src/fastoad/models/performances/mission/segments/tests/test_takeoff_segments.py

```diff
@@ -119,15 +122,14 @@ def test_rotation(polar, polar_modifier):
     assert_allclose(last_point.time, 32.58, rtol=1e-2)
     assert_allclose(last_point.mass, 69995.0, rtol=1e-4)
     assert_allclose(last_point.ground_distance, 200.64, rtol=1e-3)
-    assert_allclose(last_point.alpha, 0.134, rtol=1e-2)
```

Comment:
I see this assertion was removed, I wonder for which reason?

Reply:
Good question. It just has been lost in the process 😅 
Fixed in b824429a53462d4e4ffd0eaef3dc5e9a297181e2.